client/components/reviews/ReviewListEntry.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const ReviewListEntry = (props) => (
+  <div>
+    <div>Diner info here</div>
+      <div class="profilePhoto"></div>
```

Comment:
Since you're inside a React component, you should be using "className" to denote HTML5 classes.